Prog/AddNew/TOPA_match_notices_property.sas

```diff
@@ -61,7 +62,6 @@ proc export data=Topa_id_x_address
 run;
 
 /** address_id_ref as property id **/
-
 data Topa_addressid_merge; 
   set Topa_address_ssl_realprop_sort; 
   set Topa_id_x_address; 
```

Comment:
@eburton9 This is similar to doing a merge without a BY statement. Having two SETs will merge the based on the order of the rows in the respective data sets, but they won't necessarily match by ID. Normally you should not try to merge this way. 
But also I don't think it is necessary to merge to the real property data for what we need here.